pkg/visor/visor.go

```diff
@@ -89,7 +89,7 @@ type Node struct {
 
 	appsPath  string
 	localPath string
-	appsConf  []AppConfig
+	appsConf  map[int]AppConfig
```

Comment:
What do you think about changing the type to `map[string]AppConfig` where the key would be the app name? This way we could get an app by its name without iterating all apps.